client/my-sites/stats/stats-purchase/styles.scss

```diff
@@ -2,6 +2,7 @@
 @import "@automattic/components/src/styles/typography.scss";
 
 $button-padding: 8px 32px;
+$button-bar-wrap-width: 1346px;
```

Comment:
Adding a specific breakpoint for this issue is unreliable. Translations make it even harder to predict the size.
I got this issue for one width:
<img width="454" alt="SCR-20240118-hfvc" src="https://github.com/Automattic/wp-calypso/assets/112354940/d1a0c2c9-570b-45eb-88e7-a8d0c6d46f3b">

Try using `gap` with `flex-wrap`, but you need to remove the left margin/padding from the second element.

Reply:
Thanks! I'll give this a try today.